include/oneapi/dnnl/dnnl_types.h

```diff
@@ -2548,6 +2548,9 @@ typedef const struct dnnl_primitive *const_dnnl_primitive_t;
 /// Bias tensor argument.
 #define DNNL_ARG_BIAS 41
 
+/// Reduce tensor argument.
+#define DNNL_ARG_REDUCE 42
```

Comment:
(nit) do we envision there could be multiple reduce arguments (e.g. src and weights)?
If so, maybe put a not to reserve an index?

Reply:
Yeah internal macro would be messy. Let's keep it external with a note to reserve slots for potential extensions.